Formula/anycloud.rb

```diff
@@ -0,0 +1,37 @@
+class Anycloud < Formula
+    desc "Lambda alternative that works with multiple cloud providers"
```

Comment:
`Elastically scale webservers in any cloud provider` per [this](https://github.com/alantech/anycloud/pull/21) README update

Reply:
Done!